meet_balls_app/templates/meet_balls_app/home.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MeetBalls - Schedule Events to Find Your Ball Mates</title>
+  </head>
+  <body>
+    <header>
+      <h1>MeetBalls - Find Your Ball Mates</h1>
+      <nav>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Schedule an Event</a></li>
+          <li><a href="#">Join an Event</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>Welcome to MeetBalls</h2>
+      <p>
+        MeetBalls is an app that lets you schedule events to play with people
+        near you. Whether you're into sports, games, or just want to hang out,
+        MeetBalls makes it easy to find like-minded people to play with.
+      </p>
+      <p>With MeetBalls, you can:</p>
+      <ul>
+        <li>
+          Schedule an event for a specific activity, location, date, and time
+        </li>
+        <li>
+          Invite others to join your event and specify how many extra people you
+          want to play with
+        </li>
+        <li>
+          Chat with the other users in a group chat to coordinate and plan your
+          event
+        </li>
+        <li>
+          Browse and join other events that have been scheduled by other users
+        </li>
+      </ul>
+    </main>
+    <footer>
+      <p>Copyright © MeetBalls</p>
```

Comment:
i'm not sure if that goes hand in hand with the MIT license your repo has.

Reply:
Can you elaborate?

I can remove the copyright line..
Will that be enough?

Reply:
I don't know enough about licensing, but I'd imagine that this line might imply that you want to reserve some rights. I think the MIT [LICENSE](https://github.com/redhat-beyond/MeetBalls/blob/main/LICENSE) file that you have at the root of the repo is more permissive and practically allows any use of your code.

All in all, yes. I think removing this line will make things simpler.